### ISSUE TITLE
Problem with `jobs` option

### ISSUE THREAD

**Issue by hotrush** [overtrue/phplint#170]
```
./vendor/bin/phplint -j 20 . --exclude vendor
phplint 5.4 by overtrue and contributors.

Runtime       : 8.1.12

In OptionsResolver.php line 1029:

  The option "jobs" with value "20" is expected to be of type "int", but is of type "string".


phplint [--exclude EXCLUDE] [--extensions EXTENSIONS] [-j|--jobs JOBS] [-c|--configuration CONFIGURATION] [--no-configuration] [--no-cache] [--cache [CACHE]] [--no-progress] [--json [JSON]] [--xml [XML]] [-w|--warning] [-q|--quiet] [--no-files-exit-code] [--] [<path>...]
```

It worked fine for a long time, but started failing recently

**Comment by llaville** [comment]
I already know this problem.
I 'll give a full answer tomorrow.

**Comment by llaville** [comment]
My answer is available by #171

**Comment by llaville** [comment]
Situation is came back to normal with new releases 3.4.0, 4.5.0, 5.5.0 or 6.1.0 depending of your PHP version / platform.

This issue will be fixed for future version 7.0

**Comment by hotrush** [comment]
Works
